features/white-space-collapse.yml

```diff
@@ -2,9 +2,12 @@ name: white-space-collapse
 description: The `white-space-collapse` CSS property sets whether new line characters are shown as line breaks, and whether multiple consecutive spaces are all displayed or combined.
 spec: https://drafts.csswg.org/css-text-4/#white-space-collapsing
 group: white-space
+status:
+  compute_from: css.properties.white-space-collapse
```

Comment:
Maybe add a comment to document the fact that we're computing from this property in order to ignore the `preserve-spaces` value. And that we're doing this because `preserve-spaces` is experimental and might change (according to BCD). This will help our future selves maintain/evolve that feature over time.